esp/services/ws_packageprocess/ws_packageprocessService.hpp

```diff
@@ -104,16 +104,24 @@ class PackageMapAndSet : public CInterface, implements ISDSSubscription
         return LINK(tree);
     }
 
+    IPropertyTree *getPackageMapsOrSets(IPropertyTree *root, const char* path)
```

Comment:
@wangkx I would usually name routines like this "ensure" rather than "get".
"ensurePackageMapsOrSets".